src/pages/Leaderboard.tsx

```diff
@@ -226,7 +226,7 @@ const Leaderboard: React.FC = () => {
               }
               alt=""
             />
-            {currentStudent?.name}
+            <span id="currentStudent-name">{currentStudent?.name}</span>
```

Comment:
Follow the naming convention for ids for everywhere like this **#page-name-element-appropriate-name**